src/lm.jl

```diff
@@ -156,7 +156,24 @@ function coeftable(mm::LinearModel)
               ["x$i" for i = 1:size(mm.pp.X, 2)], 4)
 end
 
-predict(mm::LinearModel, newx::Matrix) =  newx * coef(mm)
+# Different methods of prediction for the different factorization types
+getR(x::DensePredChol) = UpperTriangular(x.chol.factors)
+getR(x::DensePredQR) = x.qr[:R]
+
+predict((::LinearModel, newx::Matrix)) = newx * coef(mm)
+
+function predict(mm::LinearModel, newx::Matrix, interval_type::Symbol, level = 0.95)
```

Comment:
Right, I am happy to change it to comply with any standards set by this community. Which software does this, though? I come from R, where the convention is to define `level` for confidence intervals, so I think of that as the most intuitive. In support of this is also naming - this is called "the 95% confidence interval", not the "0.05 confidence interval".